wire-ios/Wire-iOS/Sources/AnalytitcsServiceConfigurationBuilder.swift

```diff
@@ -23,14 +23,13 @@ struct AnalyticsServiceConfigurationBuilder {
 
     func build() -> AnalyticsServiceConfiguration? {
         guard
-            let secretKey = Bundle.countlyAppKey,
```

Comment:
should stay here, with !secretKey.isEmpty

to test prefer override in debug.xcconfig

Reply:
done